### DISCUSSION TITLE
Make related collection items as links to the actual items

### DISCUSSION THREAD

**Discussion by jaads** [payloadcms/payload#763] [Roadmap]
As an editor I would like to go to the related collection items by clicking on one of those items. 

![screenshot](https://user-images.githubusercontent.com/34009329/179422692-b593bb99-28d5-452a-b6e4-db6e986c7495.png)

**Comment by DanRibbens** [comment]
Hey @jaads 

We have this as planned, it is a good feature to add along with some other enhancements coming for relationships.

For now what you could do is use the `description` attribute that calls a function and uses values on the relationship field to dynamically add those links.

Stay tuned!
